components/camel-huaweicloud-smn/src/test/java/org/apache/camel/component/huaweicloud/smn/PublishTemplatedMessageTest.java

```diff
@@ -0,0 +1,129 @@
+package org.apache.camel.component.huaweicloud.smn;
```

Comment:
Please add the License header on all the test java files.

Reply:
License header added on all test files